### ISSUE TITLE
Why is length of audio (1.4.2) and auto updating content (2.2.2) different?

### ISSUE THREAD

**Issue by Wildebrew** [w3c/wcag#1112]
According to [WCAG 1.4.2](http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html) audio can start automatically and autoplay for 3 seconds whereas, according to [WCAG 2.2.2](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html) video or auto updating content can start automatically and last for 5 seconds.

The 3 vs. 5 second difference is hard to measure and ultimately there is no reason why these should be different.

Why would a screen reader user be less patient than someone looking at the page? These time limits should be the same. Us blind people are somewhat known for being less than patient from time to time, but I think the 2 second diffrence creates unnecessary complexity for testers.

In the bigger picture, I am not sure why auto playing audio is in the "perceivable" section whereas auto updating content is in 2.2.2, operable.

Loud audio that can't be stopped makes it impossible to operate the page just like animations that prevent a screen reader from reading the page, or a person with a cognitive impairment from perceiving the page.

Ultimately I would recommend dropping the auto playing audio success criterion altogether and adding the auto playing audio requirement to 2.2.2 (or the equivalent in WCAG AG), these are all factors that prevent users from being able to perceive or operate the webpage as a hole and should be assessed together.

**Comment by alastc** [comment]
Hi @Wildebrew,

I can ask around, but it's about 14 years since this was established so I don't know if there will be an explanation to hand.

Similarly, we have scope to add to the WCAG 2.x series but changing the time requirements would probably be a backwards-incompatible change. I.e. sites which pass WCAG 2.2 may not pass 2.0/2.1.

Unless we  increased the time on 1.4.2 perhaps? But it's been that was for a long time, what would the benefit be?

-Alastair

**Comment by Wildebrew** [comment]
I would recommend changing the 3 sseconds to 5 for 1.4.2, )maintains
backwards compatibility and places the same time limit on all auto
playing content regardless of presentation).

Standards should be simple and straight-forward. They should not
contain tiny, yet important details you have to be an expert to notice
and memorize, at least not for no reason.


I think WCAG silver provides an oppotunity for grouping all page level
animation and sound together, which to my mind is a much more logical
grouping.
Since WCAG AG is going to represent a significant departure from the
structure of WCAG I see this as an opportunity for moving this
requirement to live with its most closely related siblings.


On 4/19/20, Alastair Campbell <notifications@github.com> wrote:
> Hi @Wildebrew,
>
> I can ask around, but it's about 14 years since this was established so I
> don't know if there will be an explanation to hand.
>
> Similarly, we have scope to add to the WCAG 2.x series but changing the time
> requirements would probably be a backwards-incompatible change. I.e. sites
> which pass WCAG 2.2 may not pass 2.0/2.1.
>
> Unless we  increased the time on 1.4.2 perhaps? But it's been that was for a
> long time, what would the benefit be?
>
> -Alastair
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/w3c/wcag/issues/1112#issuecomment-616201769


-- 
Work hard. Have fun. Make history.

**Comment by bruce-usab** [comment]
I won't assert that there is a coherent argument to be made.  But back in the day, annoying sounds started immediately while annoying animations and video took a couple or three seconds to load.  Five seconds of annoying music was definitely too much.  Having both at three is logical, but at this point, I would rather not make this particular change for the 2.x series.
